tests/conftest.py

```diff
@@ -0,0 +1,82 @@
+from itertools import cycle
+
+from django.contrib.auth.models import User
+import pytest
+
+from books.models import Book, UserBook
+
+
+@pytest.fixture(scope="module")
+def books(django_db_setup, django_db_blocker):
+    with django_db_blocker.unblock():
```

Comment:
You need all 3 (`django_db_setup`, `django_db_blocker`, and `with django_db_blocker.unblock()`) to make a fixture module scope. I did this here because this setup can be re-used across tests (default = run it for each test), so it improves performance.